### ISSUE TITLE
Pop up screen - Bug or just ignore it?

### ISSUE THREAD

**Issue by vissercsp** [We-the-People-civ4col-mod/Mod#379]
[WTP Bug.docx](https://github.com/We-the-People-civ4col-mod/Mod/files/5056064/WTP.Bug.docx)

Hello Guys,

Started a new game this morning and after pressing launch this pop-up emerged. I have no idea what it is, do you...?

Kind regards,

Chris

**Comment by raystuttgart** [comment]
I hav eno idea, it might be related to some languag setting.
You are in Worldbuilder though - such informations are important.

In my installation I have no issues with Worldbuilder at all.

**Comment by vissercsp** [comment]
Hi Ray, 

Thanks for the prompt reply. I decided to do a clean install to check if the error persists. If not, I will close it later today. Otherwise, I will update you.

Cheers,

Chris

**Comment by raystuttgart** [comment]
If it persists with unmodified WTP:
- Please specify the version / release
- Please tell us exactly what you do / click

**Comment by vissercsp** [comment]
[WTP.Bug V2.docx](https://github.com/We-the-People-civ4col-mod/Mod/files/5058477/WTP.Bug.V2.docx)
Hi,

The issue is still there. Started a new custom game with WTP 2.8. I choose the Netherlands as player (Peter Stuyvesant) and checked "No wild sea animals" as an option. That is all...

Let me know if you need additional details.

Kind regards,

Chris

**Comment by raystuttgart** [comment]
Hi vissercsp,

you need to tell us such things like "I am **saving a map in Worldbuilder** and reload the map".
We are not mind readers. ;)

I have not tried anything like that in ages.

My current guess is that it might be related to "New Text and Translation System" - but again it is just a guess.
(Simply because it is complaining about character encoding as it seems.)

There have been so many changes in the last years, that it could be anything though.

**Comment by raystuttgart** [comment]
Hi guys, 

I have been **working intensively** with **Worldbuilder** when updating Maps for "Large Rivers".
Trust me, it was working **100% fine** ...

I can only suspect that the user did something that was simply not possible.
(e.g. Trying to work on an imcompatible outdated Scenario Map from an old mod version.)

If you know what you are doing with Worldbuilder however there is no bug.
(I would have noticed when I updated all these new maps.)

Updating / creating Maps in WTP (as such heavily and such extremely chaning mod) requires more modding knowledge than expected.
(Players can easily make mistakes causing bugs and warning pop-ups.)

I tried to explain Map-Making in WTP though in a thread in our forum.
